sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -318,22 +320,22 @@ export const FastFrameTemplate = html<FastFrame>`
                             <span class="sample-control-icon"></span>
                             <span class="sample-control-text">Label</span>
                             <div class="sample-control-actions">
-                                <fast-button appearance="stealth"
+                                <fast-button appearance="stealth" aria-label="stealth"
                                     >${ShareIcon}</fast-button
                                 >
-                                <fast-button appearance="stealth"
+                                <fast-button appearance="stealth" aria-label="stealth"
```

Comment:
What's the purpose of this aria-label here? I don't think this is descriptive of the intent of the button. Should this say the appearance name, or should this say what this is doing?

Reply:
Similar to the other stealth button scenario. But for this, it could say "Share button example"